Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/package.mo

```diff
@@ -28,13 +28,26 @@ T<sub>EqAirExt</sub> = T
 </p>
 <p>
 The correction term for long-wave radiation is based on the black body sky
-temperature and dry bulb temperature,</p>
+temperature and dry bulb temperature.</p>
 <p align=\"center\" style=\"font-style:italic;\">
 &Delta;T<sub>EqLW</sub> =
 (T<sub>BlaSky</sub>-T<sub>DryBul</sub>)
-&epsilon;<sub>Ext </sub>&alpha;<sub>Rad</sub>/
+&alpha;<sub>Rad</sub>/
 (&alpha;<sub>Rad</sub>+&alpha;<sub>Ext</sub>).
 </p>
+<p>
+The Guideline VDI 6007 Part 1 considers in 
+addition an environmental temperature and tilt angles for all orientations. As 
+necessary inputs for the environmental temperature are not defined in TMY weather 
+data sets, the influence of this temperature is not considered in the presented 
+models. It is in any case a minor effect as black-body sky temperature and 
+environmental temperature hardly differ. Furthermore, the Guideline VDI 6007 Part 
```

Comment:
The "hardly differ" is not true. See `"Annex60.BoundaryConditions.WeatherData.Examples.ReaderTMY3"` in which the difference between dry bulb and black body sky temperature is often around 15 Kelvin, up to 20 Kelvin.